templates/MongoDB/2/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+mongo-cluster:
+  restart: always
+  environment:
+    MONGO_SERVICE_NAME: mongo-cluster
+  tty: true
+  entrypoint: /opt/rancher/bin/entrypoint.sh
+  command:
+  - --replSet
+  - "${replset_name}"
+  image: mongo:3.2
+  labels:
+    io.rancher.scheduler.affinity: "${host_label}"
```

Comment:
Line 12 are wrong...It has to be...
    io.rancher.scheduler.affinity:host_label: ${host_label}